docs/typescript/statics-and-methods.md

```diff
@@ -80,3 +80,186 @@ const doc = new User({ name: 'test' });
 // Compiles correctly
 doc.updateName('foo');
 ```
+
+
+
+# Using `loadClass()` with TypeScript
+
+Mongoose supports applying ES6 classes to a schema using `schema.loadClass()`.
+When using TypeScript, there are a few important typing details to understand.
+
+```ts
+import { Schema, Model, model, Document } from 'mongoose';
+```
```

Comment:
I dont think this is necessary. Also in its current state, this is a somewhat undocumented codeblock.